tests/versioned/apollo-server-lambda/apollo-server-lambda-setup.js

```diff
@@ -27,7 +28,10 @@ function setupApolloServerLambdaTests({suiteName, createTests, pluginConfig}, co
     t.beforeEach(async () => {
       const { ApolloServer, gql } = require('apollo-server-lambda')
 
+      agentTesting.temporarySetEnv(t, 'NEW_RELIC_ACCOUNT_ID', 'eeeeee')
```

Comment:
i might leave a comment as to "why" this is necessary, given the other configuration items are in the config file and the linkage won't be obvious to newcomers.

Reply:
Sounds good, I'll update.